### PR TITLE
Traffic replace edson fix2

### PR DESCRIPTION
**Pull request by danielezonca**
Replace https://github.com/kiegroup/kie-wb-playground/pull/73

@kkufova @gitgabrio @porcelli

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -69,6 +77,16 @@
         <version>${project.version}</version>
         <scope>import</scope>
       </dependency>
+      <dependency>
+        <groupId>org.drools</groupId>
+        <artifactId>drools-wb-scenario-simulation-editor-api</artifactId>
```
Comment (porcelli) [inline pom.xml]: this creates a circular dependency

File: pom.xml [hunk 2]
```diff
@@ -69,6 +77,16 @@
...
+        <version>${project.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.drools</groupId>
+        <artifactId>drools-wb-scenario-simulation-editor-backend</artifactId>
```
Comment (porcelli) [inline pom.xml]: same here: this creates a circular dependency

File: traffic-violation/pom.xml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.drools</groupId>
+      <artifactId>drools-scenario-simulation-api</artifactId>
```
Comment (porcelli) [inline traffic-violation/pom.xml]: this creates a circular dependency

File: traffic-violation/pom.xml [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.drools</groupId>
+      <artifactId>drools-scenario-simulation-backend</artifactId>
```
Comment (porcelli) [inline traffic-violation/pom.xml]: this creates a circular dependency

File: pom.xml (hunk 1 above)
Comment (danielezonca) [reply pom.xml]: Removed, it is not necessary here

File: pom.xml (hunk 2 above)
Comment (danielezonca) [reply pom.xml]: Removed, it is not necessary here

File: traffic-violation/pom.xml (hunk 3 above)
Comment (danielezonca) [reply traffic-violation/pom.xml]: Are you sure? This is a module defined inside `drools` repo so I expect no circular dependency

File: traffic-violation/pom.xml (hunk 4 above)
Comment (danielezonca) [reply traffic-violation/pom.xml]: Are you sure? This is a module defined inside `drools` repo so I expect no circular dependency

File: traffic-violation/pom.xml (hunk 3 above)
Comment (porcelli) [reply traffic-violation/pom.xml]: sorry, you're correct.

File: traffic-violation/pom.xml (hunk 4 above)
Comment (porcelli) [reply traffic-violation/pom.xml]: sorry, you're correct.

**Comment by danielezonca** [comment]
@manstis 
Can you please squash and merge this?

**Comment by danielezonca** [comment]
@kiegroup/gatekeepers 
Can you please squash and merge this?

**Pull request merged**